plotting/sound.py

```diff
@@ -65,16 +65,91 @@ def plot(self):
         # Plot Sound Speed profile
         plt.subplot(gs[:, 1 if self.showmap else 0])
         ax = plt.gca()
+        i = 0
         for i, ss in enumerate(self.sspeed):
             ax.plot(ss, self.temperature_depths[i], '-')
 
         minspeed = np.amin(self.sspeed)
         maxspeed = np.amax(self.sspeed)
 
+
+        if self.query.get('annotate'):
+            # SOUND SPEED MINIMA
+            
+            minpos = np.where(self.sspeed[0] == minspeed)
+
+            # SONIC LAYER DEPTH
+
+            # Calculate
+            soniclayerdepth_value = self.sspeed[0][0:int(minpos[0])].max()
```

Comment:
I'm wondering what's the conceptual difference between `soniclayerdepth_value` and `soniclayerdepth`?

Reply:
soniclayerdepth is the depth of the local maxima, whereas soniclayerdepth_value is the sound speed at the soniclayerdepth

Reply:
Added a note to the docstring to explain this